tests/clar.h

```diff
@@ -17,9 +17,9 @@ enum cl_test_status {
 
 void clar_test_init(int argc, char *argv[]);
 int clar_test_run(void);
-void clar_test_shutdown(void);
+void clar_test_shutdown();
```

Comment:
We prefer to keep the `void` in function signatures, as it should be done for C-style method signatures. Otherwise, the function can actually accept arguments